app/models/project_source.rb

```diff
@@ -14,6 +14,8 @@ class ProjectSource < ActiveRecord::Base
   validates :source_id, uniqueness: { scope: :project_id }
   before_validation :set_account, on: :create
 
+  after_create :add_elasticsearch_data
```

Comment:
Yes Caio always need to set parent document directory as we have an issue when create parent/child on background - we have a mantis ticket with this issue